src/app/shared/object-list/duplicate-data/duplicate.model.ts

```diff
@@ -0,0 +1,42 @@
+import {autoserialize, deserialize} from 'cerialize';
+import { MetadataMap } from '../../../core/shared/metadata.models';
+import { HALLink} from '../../../core/shared/hal-link.model';
+import { CacheableObject } from '../../../core/cache/cacheable-object.model';
+import { DUPLICATE } from './duplicate.resource-type';
+import { ResourceType } from '../../../core/shared/resource-type';
+
+export class Duplicate implements CacheableObject {
```

Comment:
Please describe this class with a comment. That will help other developers understand where/how it is used

Reply:
done